### PR TITLE
[P22] Army Manager

### PR DESCRIPTION
**Pull request by ewanharris**
https://trello.com/c/ucPfBVnJ
- Adds the basic Army Manager functionality from the UML diagram
- Move some of the pre-exisiting menu items into submenus to avoid clutter

### CODE REVIEW

File: Empire - The Last Battle/Assets/Unity/Prefabs/Units/Archer.prefab.meta [hunk 1]
```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```
Comment (alasdairhurst) [inline Empire - The Last Battle/Assets/Unity/Prefabs/Units/Archer.prefab.meta:1]: Just the tip.

File: Empire - The Last Battle/Assets/Unity/Scripts/Editor/TerrainTypeDataEditor.cs [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 public class TerrainTypeDataEditor : Editor
 {
-    [MenuItem("Assets/Create/ELB/tileTypeDataBase")]
```
Comment (alasdairhurst) [inline Empire - The Last Battle/Assets/Unity/Scripts/Editor/TerrainTypeDataEditor.cs:7]: This is terrain type data though?

File: Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/Unit.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+	//TODO: Decide if this is the best way to handle an upgrade. Is it better to have a separate Upgrade class?
+	public UnitBaseData Upgrade;
+	public TileData Position;
+	public static UnitType Type;
+
+	void Start () {
+	
+	}
+
+
```
Comment (alasdairhurst) [inline Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/Unit.cs:57]: Maybe we could do with getHP, getStrength etc... which add the upgrade+basedata together if upgrade exists?

**Comment by ewanharris** [comment]
Will get round to this either tonight or tomorrow

**Comment by alasdairhurst** [comment]
:+1:

**Pull request merged**